src/main/java/com/amihaiemil/docker/LocalDocker.java

```diff
@@ -58,10 +59,19 @@ public LocalDocker(final File unixSocket){
      * @param version API version (e.g. v1.30).
      */
     public LocalDocker(final File unixSocket, final String version){
-        super(
+        this(
             new UnixHttpClient(unixSocket),
             URI.create("unix://localhost:80/" + version)
         );
     }
 
+    /**
+     * Local Docker engine.
+     * @param client The http client to use.
+     * @param baseUri Base URI.
+     */
+    LocalDocker(final HttpClient client, final URI baseUri) {
```

Comment:
@llorllale Could you accept ``String version`` here, instead of ``URI baseUri``, and move ``URI.create(...)`` in the call to ``super``? I think it's more uniform, to have all ctors accepting the version and letting the object build the URI with the needed host :D

Reply:
@amihaiemil sure